Doppler.Sap/Mappers/Billing/BillingForUsMapper.cs

```diff
@@ -124,5 +125,29 @@ public SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest b
 
             return sapSaleOrder;
         }
+
+        public SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, decimal docTotal)
+        {
+            var newIncomingPayment = new SapIncomingPaymentModel
+            {
+                CardCode = cardCode,
+                DocType = "rCustomer",
+                DocCurrency = _currencyCode,
+                TransferAccount = _transferAccount,
+                TransferSum = docTotal,
+                JournalRemarks = $"Pagos recibidos - {cardCode}",
+                PaymentInvoices = new List<SapPaymentInvoiceModel>
+                {
+                    new SapPaymentInvoiceModel
+                    {
+                        SumApplied = docTotal,
+                        DocEntry = docEntry,
+                        InvoiceType = "it_Invoice"
```

Comment:
What is "it_Invoice" ?

Reply:
Is a fixed value of SAP. the value indicates that the payment is from an invoice.